dataGenerator/s3Images.js

```diff
@@ -0,0 +1,32 @@
+const AWS = require('aws-sdk');
+AWS.config.loadFromPath('s3Config.json');
+const s3 = new AWS.S3();
+
+const params = {
+  Bucket: 'sdc-andrew-movie-posters',
```

Comment:
i would eventually extract this and ln 21 out to a config file (or `process.env` variable)

Reply:
I will look into using the dotenv package and a .env file to factor out these configuration variables!